### ISSUE TITLE
Adding a static link to the Home Menu

### ISSUE THREAD

**Issue by meduvigo** [talha131/pelican-elegant#140]
I would like to add a static link kind of <a href="file.pdf">file</a>.
I did that by adding <li><a href="file.pdf">file</a></li> to base.html file after the Home Menu button but on any update of Pelican-Elegant theme everything will be removed.

**Comment by silverhook** [comment]
@meduvigo, I apologise for the late reply, but after some hibernation, we are now picking up the pace (see #173 for details).

Can you explain your issue in a bit more detail? Or if that is not an issue any more, can you close it?

**Comment by iranzo** [comment]
@meduvigo doesn't the 'my projects' help you on this? I was thinking as of some type of 'CV in PDF' link, that can be put on a URL have the link provided on the projects list ¿?

**Comment by azzamsa** [comment]
any update on this, I think we can close this issue if no further update. I agree with @iranzo recommendation.

or

The Op can maintain his own fork, cause I think this is specific need.

**Comment by talha131** [comment]
No. This is feature is good to have. There can be number of use cases for it.

1. Link to CV.pdf
1. Link to Github profile
1. Link to LinkedIn profile

For example, I have been thinking of adding link to Elegant github repo in the top menu bar.

## How to solve it

We need to define an optional variable. It is an array of dictionaries. Each dictionary will have two keys

1. Link
1. Title

If this array is defined then we will iterate over it and place links from it in the top menu bar, right after Home menu item.

Let me know if someone has a better suggestion.
